web/src/components/Diagram/components/Timeline/Timeline.tsx

```diff
@@ -184,7 +184,7 @@ const Timeline = ({affectedSpans, spanList, selectedSpan, onSelectSpan}: IDiagra
       .append('text')
       .attr('class', 'span-duration')
       .attr('y', 30)
-      .text((d: any) => `${d.data.data?.duration} ms`);
+      .text((d: any) => d.data.data?.duration);
```

Comment:
This ticket  https://github.com/kubeshop/tracetest/issues/839 is going to introduce more of the changes related to the duration

Reply:
Cool, with this PR I believe the graphs are ok.